js-structure/js-structure_shapes/components/Pentagon/Pentagon.js

```diff
@@ -0,0 +1,10 @@
+import { getRandomColor } from "../../utils/randomColor.js";
+
+export default () => {
```

Comment:
Is not wrong per se, but the main function of a component always has a name.
Please add a name to the function.